include/deal.II/lac/utilities.h

```diff
@@ -50,6 +51,26 @@ namespace Utilities
                                       const VectorType &v0,
                                       const unsigned int k,
                                       VectorMemory<VectorType> &vector_memory);
+
+    /**
+     * Apply Chebyshev filter of @p op with a given positive @p degree
+     * to the input subspace @p x . @p a and @p b define the unwanted
+     * spectrum $ [a,b] $. @p vector_memory is used to allocate memory for
+     * temporary objects. Chebyshev polynomial will be scaled to unity at
+     * @p a_L to prevent overflow. If @p a_L equals to
+     * <code>numbers::signaling_nan<double>()</code>, no scaling is performed.
+     *
+     * @author Denis Davydov, 2017
+     */
+    template <typename OperatorType, typename VectorType>
+    void Chebyshev_filter(VectorType &x,
+                          const OperatorType &op,
+                          const unsigned int degree,
+                          const double a,
+                          const double b,
```

Comment:
`a` and `b` are not particularly descriptive variable names. They also go together. What if you encoded the unwanted spectrum as a `std::pair<double,double>` with name `unwanted_spectrum`?